examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1047 @@
+{
```

Comment:
<p>Okay first, I am curious if we should move the part of this section that I added to become a sub-section of the section titled 'Technical Background for Deutsch-Jozsa Algorithm.'</p><p> </p><p>Also, I would change this sentence 'If the function was constant then the quantum state after the H-gate is applied should be the same as it’s initial state.' to 'If the function was constant then the quantum state after the H-gates are applied should be the same as it’s initial state.'</p><p>'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Updated in most recent PR, did not move the section to Technical Background but can move it if you think it makes more sense to put it there.